ci/resources/go_expvar/test_expvar.go

```diff
@@ -38,6 +77,11 @@ func HelloServer(w http.ResponseWriter, req *http.Request) {
 }
 
 func main() {
+	// test consumer requires that garbage collection has run
+	// see foceAllocations() comment for more information
+	// an a late 2013 MacBook Pro, with go1.5.1, this takes ~1 second to run
+	forceAllocations(1000, 10000)
+
```

Comment:
Something something, the path less traveled...

![facepalm](https://i.imgur.com/iWKad22m.jpg)

A more idiomatic change has been pushed up. :smile: